shotover/src/transforms/throttling.rs

```diff
@@ -67,15 +67,20 @@ impl Transform for RequestThrottling {
         let throttled_messages: Vec<(Message, usize)> = (0..requests_wrapper.requests.len())
             .rev()
             .filter_map(|i| {
-                if self
+                match self
                     .limiter
                     .check_n(requests_wrapper.requests[i].cell_count().ok()?)
-                    .is_err()
                 {
-                    let message = requests_wrapper.requests.remove(i);
-                    Some((message, i))
-                } else {
-                    None
+                    Ok(Ok(())) => None,
+                    Ok(Err(_)) => {
+                        let message = requests_wrapper.requests.remove(i);
+                        Some((message, i))
+                    }
+                    Err(e) => {
+                        tracing::info!("{e}");
```

Comment:
accidentally left in?

Reply:
No, that error will occur when a batch is too big to go through, I figured we would want to log when that happens.

Reply:
My understanding here is that this `info` log will trigger when shotover receives a cassandra batch which contains more queries than the `max_requests_per_second` value in the topology.yaml
Such a message would never be able to make it through so I agree its probably worth communicating that it in some way.

Ideally we would include this information in the error message of the `ErrorType::Overloaded` response that goes back to the client so that the developer can easily match up the message sent to the problem.

However that would require some refactoring, so instead, for now, we could just clean up this log a bit.
I think bumping it to a warning makes sense because it indicates a misconfiguration somewhere as such a message could never have succeeded.
Lets also include some extra context since the display impl for that error wont make much sense to the user https://docs.rs/governor/latest/src/governor/errors.rs.html#10
So something like:
```rust
tracing::warn!("A message was received that could never have been successfully delivered since it contains more sub messages than can ever be allowed through via the `RequestThrottling` transforms `max_requests_per_second` configuration.")
```